queries/usersQueries.js

```diff
@@ -1,15 +1,20 @@
-const usersData = require('../usersData.json');
+const db = require('../db');
 
-const idToUserIndex = usersData.reduce((index, user) => {
-  const { id } = user;
-  index[id] = user;
-  return index;
-}, {});
+// const idToUserIndex = usersData.reduce((index, user) => {
+//   const { id } = user;
+//   index[id] = user;
+//   return index;
+// }, {});
+//new deploy
```

Comment:
At work, your team will probably have a rule against committing commented out code. Of course we all comment out code during the development process. Just be sure that before you commit your code that you check for commented out code and actually delete it.